src/useDCPWorker.tsx

```diff
@@ -355,249 +291,265 @@ function getWorkerOptionsKey() {
  *
  *  @returns {WorkerOptions?} The worker options, or null if they weren't found.
  */
-function loadWorkerOptions() {
-  let options = window.localStorage.getItem('worker-options');
-  let storage = options === null ? {} : JSON.parse(options);
+function loadWorkerOptions()
+{
+  const options = window.localStorage.getItem('dcp-worker-options');
+  const storage = options === null ? {} : JSON.parse(options);
 
   let loadedOptions;
 
-  if (Object.prototype.hasOwnProperty.call(storage, getWorkerOptionsKey())) {
+  if (Object.prototype.hasOwnProperty.call(storage, getWorkerOptionsKey()))
+  {
     loadedOptions = storage[getWorkerOptionsKey()];
-  } else if (
-    Object.prototype.hasOwnProperty.call(storage, 'defaultMaxWorkers')
-  ) {
+  }
+  else if (Object.prototype.hasOwnProperty.call(storage, 'defaultMaxWorkers'))
+  {
     loadedOptions = storage;
   }
 
-  if (!loadedOptions) return null;
+  if (!loadedOptions)
+    return null;
 
-  if (
-    Object.prototype.hasOwnProperty.call(loadedOptions, 'paymentAddress')
-  ) {
-    if (loadedOptions.paymentAddress instanceof window.dcp.wallet.Keystore) {
+  if (Object.prototype.hasOwnProperty.call(loadedOptions, 'paymentAddress'))
+  {
+    if (loadedOptions.paymentAddress instanceof window.dcp.wallet.Keystore)
+    {
       loadedOptions.paymentAddress = new window.dcp.wallet.Address(loadedOptions.paymentAddress.address);
     }
-    else if (!(loadedOptions.paymentAddress instanceof window.dcp.wallet.Address)) {
+    else if (!(loadedOptions.paymentAddress instanceof window.dcp.wallet.Address))
+    {
       loadedOptions.paymentAddress = new window.dcp.wallet.Address(loadedOptions.paymentAddress);
     }
```

Comment:
**question:** Why don't we pass the value to the constructor in either case?

The constructor accepts an instance of Address, so we can remove an unnecessary elseif statement.